tests/multisigNFTs_test.go

```diff
@@ -0,0 +1,192 @@
+		//create MultiSig Account2 : {"multisig-acc-nft-2", "multisig-acc-nft-3"}, owner=="multisig-acc-nft-1"
+		{"multiSig", false, false, "Create MultiSig Account2- Happy Path - commit  ", "multisig-acc-nft-1", "800400000cin", 0, MultisigInfo{"create", "multisig-acc-nft-1", "", 2, []string{"multisig-acc-nft-2", "multisig-acc-nft-3"}, "", 0, nil}, "MEMO : Create MultiSig Account - Happy Path", nil},
+		//create MultiSig Account3 : {"multisig-acc-nft-2", "multisig-acc-nft-3", "multisig-acc-nft-4"}, owner=="multisig-acc-nft-1"
+		{"multiSig", false, false, "Create MultiSig Account3 - Happy Path - commit ", "multisig-acc-nft-1", "800400000cin", 0, MultisigInfo{"create", "multisig-acc-nft-1", "", 2, []string{"multisig-acc-nft-2", "multisig-acc-nft-3", "multisig-acc-nft-4"}, "", 0, nil}, "MEMO : Create MultiSig Account - Happy Path", nil},
+		{"multiSig", true, true, "Create MultiSig Account - nonkyc                ", "multisig-acc-nft-1", "800400000cin", 0, MultisigInfo{"create", "multisig-acc-nft-1", "", 2, []string{"multisig-acc-nft-1", "multisig-nokyc"}, "", 0, nil}, "", nil},
+
+		//create MultiSig Account4 : {"multisig-acc-nft-2", "multisig-acc-nft-3", "multisig-acc-nft-4"}, owner=="multisig-acc-nft-1"
+		{"multiSig", false, false, "Create MultiSig Account4 - Happy Path - commit ", "multisig-acc-nft-1", "800400000cin", 0, MultisigInfo{"create", "multisig-acc-nft-1", "", 3, []string{"multisig-acc-nft-2", "multisig-acc-nft-3", "multisig-acc-nft-4"}, "", 0, nil}, "MEMO : Create MultiSig Account - Happy Path", nil},
+
+		//1. module : bank_test
+		{"bank", false, false, "top-up Multisig Group-address1 - commit", "multisig-acc-nft-1", "800400000cin", 0, bankInfo{"multisig-acc-nft-1", "nft-grp-addr-1", "10000000000cin"}, "MEMO : top-up account", nil},
+		{"bank", false, false, "top-up Multisig Group-address2 - commit", "multisig-acc-nft-2", "800400000cin", 0, bankInfo{"multisig-acc-nft-2", "nft-grp-addr-2", "10000000000cin"}, "MEMO : top-up account", nil},
+		{"bank", false, false, "top-up Multisig Group-address3 - commit", "multisig-acc-nft-3", "800400000cin", 0, bankInfo{"multisig-acc-nft-3", "nft-grp-addr-3", "10000000000cin"}, "MEMO : top-up account", nil},
+		{"bank", false, false, "top-up Multisig Group-address4 - commit", "multisig-acc-nft-4", "800400000cin", 0, bankInfo{"multisig-acc-nft-4", "nft-grp-addr-4", "10000000000cin"}, "MEMO : top-up account", nil},
+
+		//2. module : kyc_test
+		{"kyc", false, false, "Doing kyc - nft-mostafa - commit", "nft-kyc-auth-1", "0cin", 0, kycInfo{"nft-kyc-auth-1", "nft-kyc-issuer-1", "nft-kyc-prov-1", "whitelist", "nft-mostafa", "nft-mostafa", "testKyc123452222", "0"}, "", nil},
+
+		//3. module : maintenance_test
+		//add nft-mostafa as nonfungible authorised address
```

Comment:
This case doesn't have conflict with other (maintenance_tests.go)?

Reply:
Maybe better we run maintenance test first then multisig. Any idea?

Reply:
Hi, Mostafa

You are correct as this is redundant, as I planned to run this multisigNFTs_test.go as stand-alone process without involve other test-modules 1st, each time I testing new-cases.

Once fully completed this multisigNFTs_test.go later, that time only I merge with existing modules, which include the maintenance_test.go by remove the above case.

I take note your comment here.

TQ.